tests/Mf2/ParserTest.php

```diff
@@ -824,5 +824,22 @@ public function testNotMutatingPassedInDOM() {
 		Mf2\parse($inputDoc, 'http://snarfed.org/2013-10-23_oauth-dropins');
 		$this->assertEquals($refDoc, $inputDoc, 'Parsing mutated the DOMDocument.');
 	}
+
+	public function testNoImpliedURLForEmptyProperties() {
+		// In the 0.4.5 release, this caused an error
+		// https://github.com/microformats/php-mf2/issues/196
+
+		$input = <<<EOD
+<div class="h-entry">
+  <li class="h-cite u-comment">
+    <div class="vcard"></div>
+  </li>
+</div>
+EOD;
+
+		$output = Mf2\parse($input);
+		$this->assertEquals([], $output['items'][0]['properties']['comment'][0]['properties']);
+		$this->assertEquals([], $output['items'][0]['properties']['comment'][0]['children'][0]['properties']);
```

Comment:
Minor update: can use `assertEmpty()` for these.

Reply:
what does `assertEmpty` do if the value is null or not an array? I want to ensure that `properties` exists and is an array with no items.

Reply:
I think null value would pass the assertion; not sure about an empty object. [assertInternalType](https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertInternalType) can be used to verify it's an array.